tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -0,0 +1,110 @@
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { TestContext } from 'ember-test-helpers';
+import moment from 'moment';
+import { module, test } from 'qunit';
+
+import config from 'ember-get-config';
+import RegistrationModel from 'ember-osf-web/models/registration';
+import { visit } from 'ember-osf-web/tests/helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+const tagsMap: Record<string, string[]> = {
+    title: ['dc.title', 'citation_title', 'og:title', 'twitter:title'],
+    description: ['dc.abstract', 'citation_description', 'og:description', 'twitter:description'],
+    contributors: ['dc.creator', 'citation_author'],
+    affiliatedInstitutions: ['citation_author_institution'],
+    tags: ['dc.keywords'],
+    dateRegistered: ['dc.datesubmitted', 'citation_publication_date'],
+    dateModified: ['dc.datemodified'],
+    url: ['citation_public_url', 'og:url'],
+    image: ['og:image', 'twitter:image'],
+};
+
+function getMetaTagsFor(assert: Assert, prop: string): Array<string|null> {
+    const terms = tagsMap[prop];
+
+    const metaTags = terms.map(term => {
+        const k = term.includes('og:') ? 'property' : 'name';
+        const selector = `meta[${k}="${term}"]`;
+        const elements = [...document.querySelectorAll(selector)];
+        return elements.map((el: Element) => el.getAttribute('content'));
+    });
+
+    if (metaTags.length > 1) {
+        assert.ok(metaTags.every(m => JSON.stringify(m) === JSON.stringify(metaTags[0])));
+    }
+
+    return metaTags[0];
+}
+
+type RegistrationAttrs = 'title'|'description'|'dateModified'|'dateRegistered';
+
+module('Registries | Acceptance | overview.overview', hooks => {
+    setupEngineApplicationTest(hooks, 'registries');
+    setupMirage(hooks);
+
+    hooks.beforeEach(function(this: TestContext) {
+        this.store = this.owner.lookup('service:store');
+    });
+
+    test('Check head meta tags', async function(this: TestContext, assert: Assert) {
+        const reg = server.create('registration', {
+            description: 'Sun',
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+            affiliatedInstitutions: server.createList('institution', 2, {}),
+            tags: ['Social Media', 'Education'],
+            currentUserPermissions: ['admin'],
+        });
+        const contributors = server.createList('contributor', 13, { node: reg });
+
+        const registration: RegistrationModel = await this.store.findRecord('registration', reg.id, {
```

Comment:
I think it's cleaner to assert against the mirage model instance (`reg`), unless there's something you need the ember-data model instance for?